packages/browser-sync/package.json

```
@@ -48,7 +48,7 @@
     "fs-extra": "3.0.1",
     "http-proxy": "1.15.2",
     "immutable": "^3",
-    "localtunnel": "1.9.1",
+    "localtunnel": "1.9.2",
```

```suggestion
    "localtunnel": "^1.9.2",
```

How about adding a caret range? This would allow `>=1.9.2 <2.0.0`.

https://yarnpkg.com/en/docs/dependency-versions#toc-caret-ranges

In any case, would be great to get this security issue released as soon as possible, pinned or careted.

Thanks!

---

If the maintainer(s) are okay with this change it sounds reasonable. What say @shakyShane ?